examples/data-solutions/data-platform-foundations/06-common.tf

```diff
@@ -21,27 +21,53 @@ module "common-project" {
   prefix          = var.prefix
   name            = "cmn${local.project_suffix}"
   group_iam = {
+    (local.groups.data-analysts) = [
+      "roles/datacatalog.viewer",
+    ]
     (local.groups.data-engineers) = [
       "roles/dlp.reader",
       "roles/dlp.user",
       "roles/dlp.estimatesAdmin",
     ]
     (local.groups.data-security) = [
       "roles/dlp.admin",
+      "roles/datacatalog.admin"
     ]
   }
   iam = {
     "roles/dlp.user" = [
       module.load-sa-df-0.iam_email,
       module.transf-sa-df-0.iam_email
     ]
+    "roles/datacatalog.viewer" = [
+      module.load-sa-df-0.iam_email,
+      module.transf-sa-df-0.iam_email,
+      module.transf-sa-bq-0.iam_email
+    ]
+    "roles/datacatalog.categoryFineGrainedReader" = [
+      module.transf-sa-df-0.iam_email,
+      module.transf-sa-bq-0.iam_email,
+      # Uncomment if you want to grant access to `data-analyst` to all columns tagged.
+      # local.groups_iam.data-analysts
+    ]
   }
   services = concat(var.project_services, [
     "datacatalog.googleapis.com",
     "dlp.googleapis.com",
   ])
 }
 
+# Data Catalog Policy tag
+
+module "common-datacatalog" {
+  count      = var.data_catalog_tags != null ? 1 : 0
+  source     = "../../../modules/data-catalog-policy-tag"
+  name       = "${var.prefix}-datacatalog-policy-tags"
+  project_id = module.common-project.project_id
```

Comment:
project id first after source and count ;)

Reply:
Done.